Automattic/wp-calypso
# GDPR: Module/Feature Privacy Links

Ref: p1HpG7-50l-p2

Calypso: This PR adds "Privacy Information" links into each Jetpack module/feature.

<p align="center"><img src="https://user-images.githubusercontent.com/1563559/38402702-948cb006-390a-11e8-9332-96aac556938c.png" align="center" /></p>

---

<p align="center"><img src="https://user-images.githubusercontent.com/1563559/38402703-94be18c6-390a-11e8-9f72-4fabae9771dc.png" align="center" /></p>

---

<p align="center"><img src="https://user-images.githubusercontent.com/1563559/38402704-94fbca68-390a-11e8-8181-c78975716d39.png" align="center" /></p>

---

### How to Test this PR

- In Calypso Live, under Settings in the Calypso Dashboard (https://calypso.live/?branch=add/gdpr-feature-privacy-links) check all of the icons next to Jetpack features to ensure they are working properly; i.e., that the Privacy Information link is there and that it leads to the appropriate Jetpack.com privacy section.
- Help to make sure I didn't miss any Jetpack features.

### Overview of Changes

- Adds a new `SupportInfo` component. See [README](https://github.com/Automattic/wp-calypso/tree/a92cdd900ff34d7fa702b93c7193b952cee38e92/client/components/support-info) in the [`client/components/support-info`](https://github.com/Automattic/wp-calypso/tree/a92cdd900ff34d7fa702b93c7193b952cee38e92/client/components/support-info) directory. The new component is used to display a support info icon. When clicked, it displays a popover with a description, a learn more link, and a privacy info link.

  This new component is a way of making what was `.site-settings__info-link-container` into a reusable component, and therefore easier to adjust globally.

- Removes all occurrences of `.site-settings__info-link-container` in favor of the new `SupportInfo` component.

- Updates CSS adjustments across a few components to change references from `.site-settings__info-link-container` into `.support-info`.

- Adds support info icons to all `JetpackModuleToggle` instances, covering every Jetpack module/feature. **Help:** Please make sure I didn't miss any Jetpack features.

### Also In Jetpack (`/wp-admin`)

See: https://github.com/Automattic/jetpack/pull/9199
